src/main/java/de/filefighter/rest/domain/filesystem/business/FileSystemBusinessService.java

```diff
@@ -79,7 +77,7 @@ public FileSystemBusinessService(FileSystemRepository fileSystemRepository, User
                     throw new FileFighterDataException("FolderContents expected fileSystemItem with id " + fileSystemId + " but was empty.");
 
                 if (userIsAllowedToSeeFileSystemEntity(fileSystemEntityInFolder, authenticatedUser)) {
-                    String pathWithTrailingSlash = pathToFind.equals("/") ? pathToFind : pathToFind + "/";
+                    String pathWithTrailingSlash = pathToFind.equals("/") ? pathToFind : pathToFind + "/"; //NOSONAR
```

Comment:
at least put some brackets around the inline if

Reply:
okok